employee-rostering-frontend/src/store/contract/contract.test.ts

```diff
@@ -17,9 +17,10 @@
 import { mockStore } from '../mockStore';
 import { AppState } from '../types';
 import * as actions from './actions';
+import * as employeeActions from 'store/employee/actions';
 import reducer, { contractOperations } from './index';
 import { withElement, withoutElement, withUpdatedElement } from 'util/ImmutableCollectionOperations';
-import { onGet, onPost, onDelete, resetRestClientMock } from 'store/rest/RestTestUtils';
+import {onGet, onPost, onDelete, resetRestClientMock} from 'store/rest/RestTestUtils';
```

Comment:
Oddly enough, `npm lint: fix` doesn't fix the bracket spacing inconsistency (it does fix nearly all other spacing inconsistencies though).

Reply:
You need to add [object-curly-spacing](https://eslint.org/docs/rules/object-curly-spacing) rule to your `eslint.rc`

Reply:
@MusaTalluzi I'd recommend not to introduce new rule configurations as long as we can reuse ones that are part of Airbnb style. So the idea is to use the complete Airbnb style and only override minimal set of rules that are inconvenient for some reason. The groundwork has been laid in #251. Currently, `airbnb` is first and other configs override it. I chose this approach not to introduce too many rules at a time since Christopher has a lot of work in progress.

Reply:
@yurloc `object-curly-spacing` is required in [airbnb style](https://github.com/airbnb/javascript#whitespace--in-braces) but it's probably disabled by some subsequent rule/extension.

Reply:
Yes. That's what I'm saying :slightly_smiling_face: